File: 3842-toggle-light-bulbs/3842-toggle-light-bulbs.cpp
class Solution {
public:
    vector<int> toggleLightBulbs(vector<int>& bulbs) {
        int n = bulbs.size();
        vector<int> map(101, 0);

        vector<int> ans;
        for (int i = 0; i < n; i++)
            map[bulbs[i]]++;

        for (int i = 0; i < 101; i++) {
            if ((map[i] & 1) && (map[i] > 0))
                ans.push_back(i);
        }
        
        return ans;
    }
};

File: 3842-toggle-light-bulbs/3842-toggle-light-bulbs.cs
public class Solution {
    public IList<int> ToggleLightBulbs(IList<int> bulbs) {
        int n = bulbs.Count;
        var map = new bool[101];
        for (int i = 0; i < n; i++) {
            if (map[bulbs[i]])
                map[bulbs[i]] = false;
            else
                map[bulbs[i]] = true;
        }

        var ans = new List<int>();
        for (int i = 0; i < 101; i++) {
            if (map[i])
                ans.Add(i);
        }
        return ans;
    }
}